Makefile

```diff
@@ -36,7 +36,7 @@ integration_test: build_dev
 	docker compose run --rm navigator-admin-backend pytest -vvv tests/integration_tests
 
 test: build_dev
-	docker compose run --rm navigator-admin-backend pytest -vvv tests
+	docker compose run --rm navigator-admin-backend -- pytest -vvv tests --cov=app --cov-fail-under=80 --cov-report=term --cov-report=html
```

Comment:
What's the benefit of only adding here rather than on `make integration_test` and `make unit_test` as well (as that's what CI runs)?